src/libraries/System.Private.CoreLib/src/System/Reflection/SignatureHasElementType.cs

```diff
@@ -27,6 +27,7 @@ protected SignatureHasElementType(SignatureType elementType)
         public sealed override bool IsGenericTypeParameter => false;
         public sealed override bool IsGenericMethodParameter => false;
         public sealed override bool ContainsGenericParameters => _elementType.ContainsGenericParameters;
+        protected sealed override bool IsValueTypeImpl() => _elementType.IsValueType;
```

Comment:
This is implementation for arrays/pointers/byrefs. This should return `false`

Reply:
Sure no need to forward here.